pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -67,6 +67,19 @@ var OSDRequiredRoles = []string{
 	"roles/compute.admin",
 }
 
+// CCSConsoleAccessRequiredRoles is a list of Roles that a service account
+// required to get console access of CCS Accounts
+var CCSConsoleAccessRequiredRoles = []string{
```

Comment:
please see https://docs.google.com/document/d/1vqZeayNt-GUsLJslw9jTmORvUPf1WvdhRcZgZJQeCIU/edit# for the latest list of roles